templates/cloud/partners/index.html

```diff
@@ -155,10 +155,9 @@ <h2 class="muted-heading">A selection of our cloud partners</h2>
 
 {% block footer_extra %}
 <script>
-    YUI().use('event-base', function (Y) {
-        Y.on('domready', function () {
-            core.loadPartners('?programme__name=Certified%20Public%20Cloud&programme__name=Charm%20Partner%20Programme&programme__name=OpenStack&featured=true', '#cloud-partners');
-        });
-    });
+    partnerLogoClouds = [{
```

Comment:
I agree, thanks. Think this is best tackled in a follow up once this one and the local branch land. As they both touch this code.